src/components/ProductCard.vue

```diff
@@ -41,6 +48,31 @@ const oldValueTitle = ` ${t('from')} ${t('currency')} ${props.product.oldValue},
 const newValue = `${t('currency')} ${props.product.value},00`
 const selledBy = `${t('selled_by')} ${props.product.seller}`
 
+const cartStore = useCartStore();
+
+const quantityInCart = computed(() => {
+    const item = cartStore.items.find(i => i.id === props.product.id);
+    return item ? item.qtd : 0;
+});
+
+function handleAddToCart() {
+    addToCart(props.product);
+}
+
+function incrementQuantity() {
+    addToCart(props.product);
+}
+
+function decrementQuantity() {
+    const item = cartStore.items.find(i => i.id === props.product.id);
+    if (item) {
+        if (item.qtd > 1) {
+            cartStore.updateItemQuantity(item.id, -1);
+        } else {
+            cartStore.removeItem(item.id);
+        }
+    }
+}
```

Comment:
Instead of using a whole block of logic to check the item, wouldn't it be the same to replace it with this, facilitating the reading?

```suggestion
function decrementQuantity() {
  const item = cartStore.items.find(i => i.id === props.product.id);
  if (!item) return;

  if (item.qtd > 1) {
    cartStore.updateItemQuantity(item.id, -1);
  } else {
    cartStore.removeItem(item.id);
  }
}
```